src/box/xrow_io.cc

```diff
@@ -98,6 +99,7 @@ coio_read_xrow_timeout_xc(struct iostream *io, struct ibuf *in,
 void
 coio_write_xrow(struct iostream *io, const struct xrow_header *row)
 {
+	ERROR_INJECT(ERRINJ_COIO_WRITE_DROP, return);
```

Comment:
For the box.cfg.replication change I meant that it is noticed instantly in terms of timeouts: as soon as box.cfg.replication is changed, Tarantool closes the socket, without waiting for any timeouts, other side of the connection receives EOF and notices the disconnect almost immediately.

I didn't want to drop some of the packets in the connection, I wanted to drop all of them, but without closing the connection. So that I can test that disconnect timeouts are different on leader and follower when strict fencing is enabled.